statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1475,7 +1475,9 @@ def resid_working(self):
 
     @cache_readonly
     def resid_anscombe(self):
-        return self.family.resid_anscombe(self._endog, self.fittedvalues)
+        return self.family.resid_anscombe(self._endog, self.fittedvalues,
+                                          var_weights=self._var_weights,
+                                          scale=self.scale)
```

Comment:
using the estimated scale=self.scale is not backwards compatible.
Given that scale=1 in anscombe is currently enforced by unit test, I think we shouldn't change it close to a release.
Although, after figuring this out I would prefer the correctly scaled version. I was thinking about this, and for backwards compatibility we might need to work around this in another way, e.g. new resid_anscombe_scaled, or a get_resid method that allows for options.